sql/core/src/test/java/test/org/apache/spark/sql/JavaBeanDeserializationSuite.java

```diff
@@ -509,4 +514,95 @@ public void setId(Integer id) {
       this.id = id;
     }
   }
+
+  @Test
+  public void testSpark30() {
```

Comment:
what does `30` mean here?

Reply:
Actually it is just an unique id. I followed style of the test suite: https://github.com/apache/spark/blob/2e366b9ede9d079af5741d8e848def21aae3a15c/sql/core/src/test/java/test/org/apache/spark/sql/JavaBeanDeserializationSuite.java#L128

Reply:
I think that was the JIRA ticket ID?

Reply:
It seems it is. I can assign ticket number too like `testSpark27344` or nicer name like `testBeanOfLocalDateAndInstant`. I would prefer the former one.